oeo.owl

```diff
@@ -0,0 +1,230 @@
+Prefix: : <http://openenergy-platform.org/ontology/oeo/>
+Prefix: bfo: <http://purl.obolibrary.org/obo/bfo.owl#>
+Prefix: dc: <http://purl.org/dc/elements/1.1/>
+Prefix: oeo: <http://openenergy-platform.org/ontology/v0.0.1/oeo/>
+Prefix: owl: <http://www.w3.org/2002/07/owl#>
+Prefix: rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+Prefix: rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+Prefix: xml: <http://www.w3.org/XML/1998/namespace>
+Prefix: xsd: <http://www.w3.org/2001/XMLSchema#>
+
+
+
+Ontology: <http://openenergy-platform.org/ontology/oeo/>
+<http://openenergy-platform.org/ontology/v0.0.1/oeo/>
+Import: <http://purl.obolibrary.org/obo/bfo.owl>
+
+Annotations: 
+    dc:contributor "Fabian Neuhaus",
+    dc:contributor "Martin Glauer",
+    dc:contributor "Sebastian Cordes",
+    rdfs:comment "Include description of the ontology (in particular its scope) here ..."
+
+AnnotationProperty: dc:contributor
+
+    
+AnnotationProperty: definition
+
+    SubPropertyOf: 
+        rdfs:comment
+    
+    
+AnnotationProperty: rdfs:comment
+
+    
+AnnotationProperty: rdfs:isDefinedBy
+
+    
+Datatype: rdf:PlainLiteral
+
+    
+ObjectProperty: has_Input
+
+    Domain: 
+        Transformation
+    
+    Range: 
+        Data
+    
+    
+ObjectProperty: has_Objective
+
+    Domain: 
+        Optimization
+    
+    Range: 
+        ModelComponent
+    
+    
+ObjectProperty: has_Output
+
+    Domain: 
+        Transformation
+    
+    Range: 
+        Data
+    
+    
+ObjectProperty: has_part
+
+    
+ObjectProperty: has_tool
+
+    Domain: 
+        ModelCalculation
+    
+    Range: 
+        Model
+    
+    
+ObjectProperty: is_about
+
+    Domain: 
+        Paper
+    
+    Range: 
+        ModelCalculation
+    
+    
+Class: <http://purl.obolibrary.org/obo/BFO_0000002>
+
+    EquivalentTo: 
+        Continuant
+    
+    
+Class: <http://purl.obolibrary.org/obo/BFO_0000003>
+
+    
+Class: <http://purl.obolibrary.org/obo/BFO_0000015>
+
+    
+Class: <http://purl.obolibrary.org/obo/BFO_0000031>
+
```

Comment:
why are these three BFO classes repeated here without any further comment? Shouldn't they be available through the import already?

Reply:
this is a protege thing. it includes them without asking.